tests/test_result.py

```diff
@@ -79,7 +84,7 @@ def test_err_method() -> None:
 
 
 def test_no_arg_ok() -> None:
```

Comment:
@francium Sorry to intrude, but shouldn't this test case gets removed or expects a raise now the No Argument case of `Ok` is removed?

Reply:
You're right. Thanks for pointing this out